src/repository/migrations/0033_file_upload_texts.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.16 on 2023-02-22 13:57
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0078_auto_20230222_1317'),
```

Comment:
Missing a dependency migration:

```shell
django.db.migrations.exceptions.NodeNotFoundError: Migration repository.0033_file_upload_texts dependencies reference nonexistent parent node ('core', '0078_auto_20230222_1317')
```

Reply:
> Missing a dependency migration:
> 
> ```shell
> django.db.migrations.exceptions.NodeNotFoundError: Migration repository.0033_file_upload_texts dependencies reference nonexistent parent node ('core', '0078_auto_20230222_1317')
> ```

The dependencies for this migration are super weird. It shouldn't require this core migration or the press one. I've removed this. I've also added an example of what it looks like:

![image](https://github.com/BirkbeckCTP/janeway/assets/8321378/c9f037a1-fd4f-45f8-9e51-a5918b31ff19)

Reply:
Thanks. It is now saying this needs merging with others on 1.5: `CommandError: Conflicting migrations detected; multiple leaf nodes in the migration graph: (0033_file_upload_texts, 0035_alter_preprintaccess_options in repository).`

Reply:
Egads. Should be okay now.